db/re/status.yml

```diff
@@ -72,8 +72,10 @@ Body:
       Stun: true
       Sleep: true
       Burning: true
+      Protection: true
     EndOnStart:
       Aeterna: true
+      Protection: true
```

Comment:
Why did you make `sc_protection` end when `sc_stone` starts since `sc_stone` should fail when `sc_protection` is enabled? for forced `sc_stone`?

Reply:
Sorry I can't understand your question. My intend is to end `sc_stone` when `sc_protection` get started.

Reply:
`EndOnStart: List of status that will end if the status activates.`

For the case of `sc_stone`, `sc_protection` end when `sc_stone` activates because of EndOnStart.